fe/fe-core/src/main/java/org/apache/doris/mtmv/MTMVPlanUtil.java

```diff
@@ -60,6 +60,21 @@ public static ConnectContext createMTMVContext(MTMV mtmv) throws AnalysisExcepti
         return ctx;
     }
 
+    public static ConnectContext createMTMVContext(long ctlId, long dbId) throws AnalysisException {
```

Comment:
There is some overlap with the logic of createMTMVContext (MTMV mtmv), abstract it a bit